.github/workflows/ci.yml

```diff
@@ -32,6 +32,31 @@ jobs:
         shell: bash
         run: mvn test -B
 
+  early_access:
+    name: 'JDK Early-Access'
+    continue-on-error: true
```

Comment:
This doesn't prevent the workflow from being marked as a failed. See https://github.com/actions/toolkit/issues/399 for details

Reply:
According to [continue-on-error](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error)'s documentation, it should work... 🤔

Reply:
Hmm. Is `continue-on-error` definitely supported for the entire `job`, or does it need to be on the affected `steps`? I saw a few examples of the latter looking at other github actions configs. Although, from https://github.blog/changelog/2020-04-15-github-actions-new-workflow-features/ it definitely sounds like it's supposed to work on the job.

Reply:
It did work as expected on the branch of my forked repository. See https://github.com/google/google-java-format/issues/538#issuecomment-733933661

Just this PR was marked as failed -- perhaps it would work after merging to master.

Reply:
Ah, thanks for clarifying. I'll go ahead and merge this.